data/src/main/java/pl/edu/agh/model/Project.java

```diff
@@ -1,5 +1,36 @@
 package pl.edu.agh.model;
 
+import java.util.List;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Transient;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Entity
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
 public class Project {
 
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private long id;
```

Comment:
private Long id;
przynajmniej tak uzgodniliśmy z @szbrnl w innym pull requescie
ale malo wazne bo i tak bedzie zmienione na common base class

Reply:
przynoszę oświaty kaganiec:
"We recommend that you declare consistently-named identifier properties on persistent classes and that you use a nullable (i.e., non-primitive) type."

http://docs.jboss.org/hibernate/core/3.3/reference/en/html/persistent-classes.html#persistent-classes-pojo-identifier

Reply:
nie wiem jak to dokładnie teraz wygląda, ale kiedyś w hibernate była taka metoda `saveOrUpdate`, która aktualizowała w bazie obiekty mające jakieś id, a te bez id zapisywała i nadawała im id zwrócone przez bazę